### ISSUE TITLE
SubScript/SuperScript in cell value

### ISSUE THREAD

**Issue by SravaniBeeram** [ExcelDataReader/ExcelDataReader#497]
Hi everyone,

Is there a way to ignore subscript/superscript in a cell value? or a metadata notifying if their existence?

**Comment by andersnm** [comment]
Hi,

Am I assuming correctly if you've got a cell with rich text, like for example "See footnote <sup>[1]</sup>", and ExcelDataReader returns this as "See footnote[1]" and you want to remove the part with "[1]" from it?

There is no deterministic way to do this at the moment. ExcelDataReader removes all rich text formatting from the cell contents, so I presume a fix involves returning the formatted rich text cell contents, and leave it to the developer(you) how to postprocess and extract the relevant data. Admittedly, this feature is not something we're planning to do anytime soon.

**Comment by SravaniBeeram** [comment]
Yes, correct - it returns "See footnote[1]". In our data, we can't tell apart if its data or footnotes. 

Thank you for the update.
